src/components/HomePage/AccountCard/AccountCard.container.ts

```diff
@@ -1,27 +1,23 @@
-import { getNetworks } from 'decentraland-dapps/dist/modules/wallet/selectors'
+import { connect } from 'react-redux'
+
 import { openModal } from 'decentraland-dapps/dist/modules/modal/actions'
 import { Networks } from 'decentraland-dapps/dist/modules/wallet'
-import { connect } from 'react-redux'
-import { RootState } from '../../../modules/reducer'
+import { getNetworks } from 'decentraland-dapps/dist/modules/wallet/selectors'
+
 import AccountCard from './AccountCard'
-import {
-  MapDispatch,
-  MapDispatchProps,
-  MapStateProps,
-  OwnProps,
-} from './AccountCard.types'
+import { MapDispatch, MapDispatchProps, MapStateProps, OwnProps } from './AccountCard.types'
+import { RootState } from '../../../modules/reducer'
```

Comment:
I think that there might be a problem with the import order. './AccountCard.types' must be lower in order. The same goes for `AccountCard`.